File: CPP/Program/Clothing.cpp
#include <iostream>
#include <string>
#include "Product.cpp"
using namespace std;

class Clothing : public Product
{
    private: 
        string size;
        string materials;
        string gender;

    public:
        Clothing()
        {
            this->size = "";
            this->materials = "";
            this->gender = "";
        }

        Clothing(string size, string materials, string gender)
        {
            this->size = size;
            this->materials = materials;
            this->gender = gender;
        }

        string get_size() const
        {
            return this->size;
        }

        void set_size(string size)
        {
            this->size = size;
        }

        string get_materials() const
        {
            return this->materials;
        }

        void set_materials(string materials)
        {
            this->materials = materials;
        }

        string get_gender() const
        {
            return this->gender;
        }

        void set_gender(string gender)
        {
            this->gender = gender;
        }

        ~Clothing()
        {

        }

};

File: Python/Program/Clothing.py
from Product import Product

class Clothing(Product):
    def __init__(self, size="", materials="", gender="", id=0, name="", brand="", price=""):
        super().__init__(id, name, brand, price)
        self.size = size
        self.materials = materials
        self.gender = gender

    def get_size(self):
        return self.size

    def set_size(self, size):
        self.size = size

    def get_materials(self):
        return self.materials

    def set_materials(self, materials):
        self.materials = materials

    def get_gender(self):
        return self.gender

    def set_gender(self, gender):
        self.gender = gender
